taichi/runtime/llvm/llvm_context.h

```diff
@@ -144,14 +136,15 @@ class TaichiLLVMContext {
       std::string name,
       llvm::LLVMContext *context = nullptr);
 
-  void add_function_to_snode_tree(int id, std::string func);
-
-  void delete_functions_of_snode_tree(int id);
+  void delete_snode_tree(int id);
 
   void add_struct_for_func(llvm::Module *module, int tls_size);
 
   static std::string get_struct_for_func_name(int tls_size);
 
+  std::unique_ptr<LLVMCompiledData> link_compile_data(
```

Comment:
```suggestion
  std::unique_ptr<LLVMCompiledData> link_compiled_data(
```

Reply:
maybe we can call it `link_compiled_tasks`

Reply:
sg!